topics/transcriptomics/tutorials/mirna-target-finder/slides.html

```diff
@@ -0,0 +1,166 @@
+---
+layout: tutorial_slides
+logo: "GTN"
+
+title: "Whole transcriptome analysis of arabidopsis Thaliana"
+contributors:
+  - gallardoalba
+  - pavanvidem
+
+priority: 4
+
+zenodo_link: ""
+
+
+---
+
+# (m)RNA-Seq data analysis
+
+???
+- (m)RNA-Seq data analysis
+
+---
+
+# Experimental design
+
+![Experimental deign](../images/plant_rnaseq_pipeline2.png)
+
+???
+- The second part of the tutorial deals with the mRNA-Seq data analysis and micro RNA target prediction.
+- First, we assess the quality of the mRNA-Seq reads, then quantify the genes, and carry out differential gene expression analysis.
+- Finally, we will predict which mRNAs are possibly downregulated by the upregulated micro RNAs.
+
+---
+
+# Background on data and tools
+
+???
+- Now we will look at  the data and tools that we are going to use in the tutorial
+
+---
+
+## Background on mRNA-Seq data
+
+![mRNA-Seq data used in the analysis](../images/data_plant_mrna.png)
+
+???
+- The mRNA-Seq data were generated using Illumina HiSeq 2000 sequencing platform.
+- Similar to micro RNA-Seq data, we have data from two conditions, namely control and brassinolide treatment.
+- And we have two biological replicates from each condition.
+.footnote[[*Oh E, Zhu JY, Bai MY, Arenhart RA et al., Elife, 2014*](https://www.ncbi.nlm.nih.gov/pubmed/24867218)]
+
+---
+
+## Tools used in mRNA-Seq data analysis
+
+![Tools used for the analysis](../images/tools_plant_mrna.png)
+
+???
+- We start with the quality assessment of the raw sequencing reads using Fast QC and Multi QC.
```

Comment:
FastQC and MultiQC?

Reply:
I don't know how Polly reads it. This is why I wrote them separated.

Reply:
Then those changes can be included after the recording :)

Reply:
ah true, that makes sense, ignore my comments then and let's make Polly happy :)